packages/server/graphql/intranetSchema/mutations/enableSAMLForDomain.ts

```diff
@@ -19,13 +19,18 @@ const enableSAMLForDomain = {
     const r = await getRethink()
     const normalizedDomain = domain.toLowerCase()
     const normalizedUrl = url.toLowerCase()
+
     await r
       .table('SAML')
-      .insert({
-        domain: normalizedDomain,
-        url: normalizedUrl,
-        metadata: metadata
-      })
+      .insert(
+        {
+          id: normalizedDomain,
+          domain: normalizedDomain,
```

Comment:
@mattkrick as you mentioned that we're still using `domain`, I've re-added it to the mutation insert. This way we can still do an atomic upsert while keeping the domain field. Is this what you had in mind?
I've just pushed an update to the mutation and migration following your feedback